### ISSUE TITLE
Context menu not usable on mobile wide screen

### ISSUE THREAD

**Issue by mr-manuel** [grafana/grafana#52010]
**What happened**:
I‘m trying to edit a panel in a dashboard, the context menu opens, but a touch on any element has no action.

![32A39608-7D7F-43B9-A5D6-58147B8F0EB2](https://user-images.githubusercontent.com/12592765/178097144-ab3f57f5-d3de-4a6b-9c1f-66cd3c81fe48.jpeg)

**What you expected to happen**:
The context menu should also work on iPadOS. On iOS it works fine.

**How to reproduce it (as minimally and precisely as possible)**:
Use an iPad, open a dashboard and try to edit a panel.

**Anything else we need to know?**:
I tried different browsers on iPadOS like Safari, Firefox, Brave, Google Chrome and Edge. All have the same behavior.

**Environment**:
- Grafana version: 9.0.2 and earlier
- User OS & Browser: iOS 15.6, 15.5 and earlier, latest Safari, Firefox, Brave, Google Chrome and Edge

**Comment by bboreham** [comment]
> I tried different browsers on iPadOS

Unfortunately Apple does not allow different browser implementations: they are all Safari with different wrapping.

**Comment by natellium** [comment]
Hey @mr-manuel thanks for reporting this issue. I was able to reproduce the bug, but it seems to only happen in **landscape mode**.

**Comment by tolzhabayev** [comment]
@natellium most likely related to https://github.com/react-grid-layout/react-draggable/issues/666

**Comment by mr-manuel** [comment]
@natellium indeed in portrait mode it works

**Comment by natellium** [comment]
Thanks @mr-manuel. I'll move this into our backlog. I understand it can be annoying but as there's a workaround I don't we'll have an ETA. Nevertheless I'll label it as help-wanted in case someone from the community wants to jump on this :)

**Comment by mr-manuel** [comment]
Better a workaround than nothing 😄 many thanks

**Comment by spincemaille** [comment]
Hi, on my iPadOS devices (ipad mini last ben & iPad Pro m1), even in portrait it doesn’t work. Any update on a final fix ?

**Comment by matejgordon** [comment]
Hello, it's not working on iPad Air 4 and 5 as well.
Any update on this topic?

**Comment by Muegic** [comment]
Same problem on Safari (iPadOS 16.3) - BUT it does work flawlessly on Brave browser and Chrome.
Good job Apple…

**Comment by LarsStegman** [comment]
That's interesting, as Brave and Chrome are both using the Webkit engine as well on iOS.

**Comment by Muegic** [comment]
> Same problem on Safari (iPadOS 16.3) - BUT it does work flawlessly on Brave browser and Chrome, even in landscape.

Update two days later: I was able to edit panels on Brave - for one session. No reactions to any touches to the panel name/context menu since then. 
Then I tried Opera, context menu/editing panels worked fine for one session/day and then stopped working.
Now I‘m on Chrome, logged in and it‘s working. I‘ll bet, when I come back tomorrow it doesn’t work anymore.

I also noticed, that when I‘m logged out, the context menu can be used on any iPad browser without any issues. But once you logged in, it‘s over… except for the first session ??
That’s so strange. Can someone confirm?

Btw portrait or landscape made no difference for me.
